operator_ui/src/components/Bridges/Form.tsx

```diff
@@ -139,9 +139,10 @@ Form.defaultProps = {
 
 const WithFormikForm = withFormik<OwnProps, FormValues>({
   mapPropsToValues({ name, url, minimumContractPayment, confirmations }) {
-    const shouldPersist = Object.keys(get('persistBridge')).length !== 0
-    const persistedJSON = shouldPersist && get('persistBridge')
-    if (shouldPersist) set('persistBridge', {})
+    const shouldPersist =
```

Comment:
this looks similar to the below block on 84 of Form.js, should it be its own function?

Reply:
I've pulled these changes out into a separate PR as they are unrelated to the original story https://github.com/smartcontractkit/chainlink/pull/1922

Reply:
Mind putting them on this PR? Should be tiny and good use of boy scout principle.

Reply:
@j16r done